documentation/Add-PnPFileSensitivityLabel.md

```diff
@@ -0,0 +1,100 @@
+---
+Module Name: PnP.PowerShell
+schema: 2.0.0
+applicable: SharePoint Online
+online version: https://pnp.github.io/powershell/cmdlets/Add-PnPFileSensitivityLabel.html
+external help file: PnP.PowerShell.dll-Help.xml
+title: Add-PnPFileSensitivityLabel
+---
+
+# Add-PnPFileSensitivityLabel
+
+## SYNOPSIS
+
+**Required Permissions**
+
+  * Microsoft Graph API : One of Files.ReadWrite.All, Sites.ReadWrite.All
+
+Add the sensitivity label information for a file in SharePoint.
+
+## SYNTAX
+```powershell
+Add-PnPFileSensitivityLabel -Url <String> -SensitivityLabelId <Guid> -AssignmentMethod <Enum> -JustificationText <string>
+```
+
+## DESCRIPTION
+
+The Add-PnPFileSensitivityLabel cmdlet adds the sensitivity label information for a file in SharePoint using Microsoft Graph. It takes a URL as input, decodes it, and specifically encodes the '+' character if it is part of the filename. It also takes the sensitivity label Id , assignment method and justification text values as input.
+
+## EXAMPLES
+
+### Example 1
+This example adds the sensitivity label information for the file at the specified URL.
+
+```powershell
+Add-PnPFileSensitivityLabel -Url "/sites/Marketing/Shared Documents/Report.pptx" -SensitivityLabelId b5b11b04-05b3-4fe4-baa9-b7f5f65b8b64 -JustificationText "Previous label no longer applies" -AssignmentMethod Privileged
+```
+
+This example adds the sensitivity label information for the file at the specified URL.
+
```

Comment:
Can we have an example of how to remove the label as well, as that requires an empty string as I read?

Reply:
Added another example to remove the label